doctr/models/classification/vit/tensorflow.py

```diff
@@ -38,6 +38,17 @@
 }
 
 
+class GELU(layers.Layer):
+    """Gaussian Error Linear Unit activation function"""
+
+    def __init__(self, approximate: bool = False, **kwargs: Any) -> None:
+        super().__init__(**kwargs)
+        self.approximate = approximate
+
+    def call(self, x: tf.Tensor) -> tf.Tensor:
+        return gelu(x, approximate=self.approximate)
```

Comment:
Using `https://www.tensorflow.org/api_docs/python/tf/keras/layers/Activation` with `tf.nn.gelu` should work, no ?

Reply:
Good catch i will test it but it should work